lib/adapters/fs.js

```diff
@@ -14,7 +15,7 @@ module.exports = objectAssign({}, FS, {
 
   writeFile: fs.writeFile,
 
-  mkdir: fs.mkdir,
+  mkdir: mkdirp,
```

Comment:
Ok. I created an issue #69 to change the integration tests to use `tmp` and to add an integration test with a recursive `rootDirectory`. For now, can I merge this? @valentin-radulescu-hs